### PR TITLE
added aditional content to foundation/Choose Your Path Forward

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Because

i have added additional content to foundation/Choose Your Path Forward,
article which i have added  (https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/)

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by sumitshinde-84** [comment]
request for review @01zulfi

File: foundations/tying_it_all_together/conclusion.md [hunk 1]
```diff
@@ -56,4 +56,4 @@ The journey has just begun!
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+* If you are still confuse, Checkout this article to get more facts, prons and cons about both languages [Cloud-Infrastucter- service article](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```
Comment (CouchofTomato) [inline foundations/tying_it_all_together/conclusion.md:59]: Can we correct the typos here please.

 If you are still confused, Checkout this article to get more facts, pros and cons about both languages [Ruby on Rails vs Javascript](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).

**Review by CouchofTomato — CHANGES_REQUESTED** [review]

File: foundations/tying_it_all_together/conclusion.md [hunk 2]
```diff
@@ -56,4 +56,4 @@ The journey has just begun!
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+* If you are still confuse, Checkout this article to get more facts, prons and cons about both languages [Ruby on Rails vs Javasctipt.](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```
Comment (CouchofTomato) [inline foundations/tying_it_all_together/conclusion.md:59]:
```suggestion
* If you are still confused, Check out this article to get more facts, pros and cons about both languages [Ruby on Rails vs Javasctipt.](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```

**Review by CouchofTomato — COMMENTED** [review]
Thanks @sumitshinde-84 

A couple of changes weren't actioned and I noticed a couple more. Hopefully it's clearer now what needs to change.

**Comment by sumitshinde-84** [comment]
@CouchofTomato  yes sir i have made changes , as you suggest me .

File: foundations/installations/computer_basics.md [hunk 3]
```diff
@@ -40,3 +40,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 * Checkout [GCF's page on "What are accessibility features"](https://edu.gcfglobal.org/en/computerbasics/using-accessibility-features/1/) to learn how you can leverage these features to use your computer efficiently.
+
```
Comment (CouchofTomato) [inline foundations/installations/computer_basics.md:43]: We don't need this extra space here.

File: foundations/installations/computer_basics.md [hunk 4]
```diff
@@ -40,3 +40,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
 * Checkout [GCF's page on "What are accessibility features"](https://edu.gcfglobal.org/en/computerbasics/using-accessibility-features/1/) to learn how you can leverage these features to use your computer efficiently.
```
Comment (CouchofTomato) [inline foundations/installations/computer_basics.md:42]:
```suggestion
- Checkout [GCF's page on "What are accessibility features"](https://edu.gcfglobal.org/en/computerbasics/using-accessibility-features/1/) to learn how you can leverage these features to use your computer efficiently.
```

File: foundations/installations/computer_basics.md [hunk 5]
```diff
@@ -40,3 +40,5 @@ This section contains questions for you to check your understanding of this less
...
+
+* If you feel like spending more time learning about this topic, check out [FreeCodeCamp's excellent course on computer basics](https://youtu.be/y2kg3MOk1sY)
```
Comment (CouchofTomato) [inline foundations/installations/computer_basics.md:44]:
```suggestion
- If you feel like spending more time learning about this topic, check out [FreeCodeCamp's excellent course on computer basics](https://youtu.be/y2kg3MOk1sY)
```

**Review by CouchofTomato — CHANGES_REQUESTED** [review]
Hey @sumitshinde-84 

I'm really sorry to keep throwing this back at you. These are the last changes 🙏

File: foundations/tying_it_all_together/conclusion.md [hunk 6]
```diff
@@ -56,4 +56,4 @@ The journey has just begun!
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- If you are still confused, Check out this article to get more facts, pros and cons about both languages [Ruby on Rails vs Javasctipt.](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```
Comment (CouchofTomato) [inline foundations/tying_it_all_together/conclusion.md:59]:
```suggestion
- If you are still confused, Check out this article to get more facts, pros and cons about both languages [Ruby on Rails vs Javascript.](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```

File: foundations/installations/computer_basics.md (hunk 3 above)
Comment (CouchofTomato) [reply foundations/installations/computer_basics.md:43]: We still need to remove this extra space.

**Review by CouchofTomato — CHANGES_REQUESTED** [review]
Hey @sumitshinde-84 

I noticed a typo which was my fault.

However I need that extra space on line 43 of computer_basics to be removed please as well.

**Comment by sumitshinde-84** [comment]
Hello  sir ,
I have made changes as you suggest me  if everything is ok then please merge it

**Review by CouchofTomato — APPROVED** [review]

**Comment by sumitshinde-84** [comment]
@CouchofTomato  , 
Hii Austin sir,
I am sumit from India and I have learned frontend development from odin,
It was amazing experience learning through Odin but now I wanna be member of odin organisations , being part of odin project I will learn lot of precious thing and also I will get opportunity to contribute in my School of programming .

**Pull request merged**